challenges/qr_code_component/index.html

```diff
@@ -2,62 +2,65 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>
-      QR code component &bullet; Frontend Mentor Challenges &bullet; Optimum BH
-      Internship
-    </title>
-    <link
-      rel="apple-touch-icon"
-      sizes="180x180"
-      href="../images/apple-touch-icon.png"
-    />
     <link
       rel="icon"
       type="image/png"
       sizes="32x32"
       href="../images/favicon-32x32.png"
     />
+    <link href="style.css" rel="stylesheet" />
     <link
-      rel="icon"
-      type="image/png"
-      sizes="16x16"
-      href="../images/favicon-16x16.png"
+      href="https://fonts.googleapis.com/css2?family=Outfit:wght@400;700&display=swap"
+      rel="stylesheet"
     />
-    <link rel="manifest" href="../site.webmanifest" />
-    <link rel="stylesheet" href="style.css" />
+    <title>
+      QR code component • Frontend Mentor Challenges • Optimum BH Internship
+    </title>
   </head>
-  <body class="m-0">
+  <body
+    class="font-display text-dark-blue bg-light-gray m-0 grid h-screen place-items-center overflow-hidden text-base font-normal md:text-lg"
+  >
     <div
-      class="flex h-[100vh] w-full flex-col place-content-center text-center"
+      class="mx-auto grid w-3/4 max-w-sm grid-flow-row place-items-center gap-3 rounded-lg bg-white px-4 pt-4 pb-8 text-center"
     >
-      <div>
+      <div class="overflow-hidden rounded-lg">
+        <img
+          src="../images/image-qr-code.png"
```

Comment:
Image path is incorrect. Check the saved screenshots - they aren't showing the image..

Reply:
I have fixed the image path.